cirq/google/api/v2/results_test.py

```diff
@@ -2,8 +2,8 @@
 import pytest
 
 import cirq
-from cirq.api.google.v2 import result_pb2
 from cirq.google.api import v2
+from cirq.google.api.v2 import result_pb2
```

Comment:
Alternately, remove this and change to `v2.result_pb2` in the code.

Reply:
Done.